nixos/modules/services/security/cfssl.nix

```diff
@@ -152,6 +152,31 @@ in
       description = "Certificate db configuration file. Path must be writeable.";
     };
 
+    disable = lib.mkOption {
+      default = [];
+      type = lib.types.listOf lib.types.str;
+      description = ''
+        Endpoints to disable.
+
+        Even if authentication is enabled, some endpoints don't require
+        authentication and allow issuing certificates unauthorized.
+
+        https://github.com/cloudflare/cfssl/issues/250
```

Comment:
I did try this with the default kubernetes `easyCerts` setup and did manage to get full kubernetes admin from a local user via the insecure endpoint.